modules/kmm-running-depmod.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-running-depmod_{context}"]
+
+= Running `depmod`
```

Comment:
[Do not use backticks or other markup in assembly or module headings.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)

Reply:
Removed backticks.